tests/core/editor/custom_config_race.js

```diff
@@ -1,41 +1,66 @@
 /* bender-tags: editor */
 
-var assetsPath = '%TEST_DIR%_assets/',
-	instances = {},
-	initEditors = ( function() {
-		var count = 0;
-
-		return function( editors, callback ) {
-			var total = CKEDITOR.tools.object.keys( editors ).length;
-
-			for ( var e in editors ) {
-				instances[ e ] = CKEDITOR.replace( e,
-					CKEDITOR.tools.extend( editors[ e ], {
-						plugins: 'wysiwygarea',
-						on: {
-							instanceReady: function() {
-								if ( ++count == total )
-									callback();
+var assetsPath = '%TEST_DIR%_assets/';
+
+function initEditors( editors ) {
+	var count = 0;
+	var instances = {};
+
+	return function( callback ) {
+		var total = CKEDITOR.tools.object.keys( editors ).length;
+
+		for ( var e in editors ) {
+			instances[ e ] = CKEDITOR.replace( e,
+				CKEDITOR.tools.extend( editors[ e ], {
+					plugins: 'wysiwygarea',
+					on: {
+						instanceReady: function() {
+							if ( ++count == total ) {
```

Comment:
~Someone once told me that using increment operator is good only in `for` loop declaration as it decreases the readability everywhere else, WDYT? 😄~ Now I see you just outdented this part of code so my joke was inappropriate 🤷‍♂️ Still, maybe let's switch it to a simple
```js
count++;

if ( count == total ) {
```
for the sake of future CKE4 maintainers.

Reply:
It's really nice to see how sides turned out! Indeed I didn't pay enough attention to leave this code better then I found it.